python/foglamp/services/core/api/snapshot/table.py

```diff
@@ -67,7 +68,7 @@
     except (StorageServerError, Exception) as ex:
         raise web.HTTPInternalServerError(reason='{} table snapshot could not be created. {}'.format(table, str(ex)))
     else:
-        return web.json_response(retval)
+        return web.json_response(retval["created"])
```

Comment:
Better to handle like below
```
result = retval["created"]
......
except KeyError as ex:
        raise web.HTTPNotFound(reason='{}........'.format(str(ex)))
else:
      return web.json_response(result)

```
Also same for retval["loaded"], ["deleted"] etc.

Reply:
Yes, this is yet another valid approach. But let us stick to on relying upon StorageServerError for storage layer errors, for the sake of uniformity, as we have been doing at other places.